package.json

```diff
@@ -1,12 +1,13 @@
 {
-  "name": "spdx-satisfies",
+  "name": "@one-beyond/spdx-license-satisfies",
   "description": "test whether SPDX expressions satisfy licensing criteria",
-  "version": "5.0.1",
+  "version": "1.0.0",
   "author": "Kyle E. Mitchell <kyle@kemitchell.com> (https://kemitchell.com)",
```

Comment:
IMO, we can remove original author and contributors and we can say in the README file that where the original package is coming from. Because  it may be confusing that this new package says that the author is the same one as the original one.

Apart form that, the repo itself says this is a fork

Reply:
+1

Reply:
Ok. What author should we give it? README already says it's a fork

Reply:
I removed both